airflow-core/src/airflow/assets/manager.py

```diff
@@ -229,12 +242,39 @@ def notify_asset_alias_created(asset_assets: AssetAlias):
     def notify_asset_changed(asset: Asset):
         """Run applicable notification actions when an asset is changed."""
         try:
+            # TODO: AIP-76 this will have to change. needs to know *what* happened to the asset (e.g. partition key)
+            #  maybe we should just add the event to the signature
+            #  or add a new hook `on_asset_event`
             get_listener_manager().hook.on_asset_changed(asset=asset)
         except Exception:
             log.exception("error calling listener")
 
     @classmethod
-    def _queue_dagruns(cls, asset_id: int, dags_to_queue: set[DagModel], session: Session) -> None:
+    def _queue_dagruns(
+        cls,
+        asset_id: int,
```

Comment:
```suggestion
        cls,
        *,
        asset_id: int,
```

since we only call it with keyword argument, not a bad idea to make it keyword only

Reply:
ok